nautobot/ipam/factory.py

```diff
@@ -439,7 +422,6 @@ class Params:
     location = factory.Maybe(
         "has_location", random_instance(lambda: Location.objects.get_for_model(Prefix), allow_null=False), None
     )
-    role = factory.Maybe("has_role", random_instance(Role), None)
```

Comment:
this needs to change to use Roles from the extras factory, instead of being removed.